src/objects.h

```diff
@@ -483,7 +483,11 @@ class ByteArray : public HeapObject {
      return static_cast<ByteArray*>(byte_array);
   }
 
-  void resize(int new_length);
+  // Only for external byte arrays that were malloced.  Does not change the
```

Comment:
Would it be hard to change the accounting? I fear that if we don't, we will end up we will end up with really messed up accounting numbers and this will definitely lead to assertion failures and unexpected behavior.